rest_framework_tracking/mixins.py

```diff
@@ -120,17 +120,23 @@ def _clean_data(self, data):
         You can define your own sensitive fields in your view by defining a set
         eg: sensitive_fields = {'field1', 'field2'}
         """
-        data = dict(data)
+        # data = dict(data)
```

Comment:
This line is needed (maybe not there) otherwise you'll modify the original values which you don't want

Reply:
for me this was giving error that's why I commented the code.

Reply:
Yes I totally understand but this is critical, if you don't do it, you will have other errors. Again you don't want to modify the data you send to your API